package.json

```diff
@@ -37,6 +37,8 @@
     "@apollo/client": "^3.7.10",
     "@apollo/space-kit": "^9.11.0",
     "@emotion/react": "^11.11.1",
+    "@headlessui/react": "^1.7.17",
```

Comment:
Curious your thoughts on whether we should standardize around [Radix](https://www.radix-ui.com/) or [headless ui](https://headlessui.com/). We are using the `@radix-ui/react-tabs` component, but might be good to avoid the mix & match strategy for the "headless" components.

I don't have a strong opinion on direction or the other. I use Radix in the Spotify UI and I like it, but from the documentation on each they seem pretty similar. If we go this route, I'd ask that we migrate the tabs component over to `@headlessui/react` to keep it all in the same library.